docs/reference/profiler.md

```diff
@@ -1,3 +1,4 @@
 # Computational Performance Profilers
+Profilers are responsible for generating benchmarking metrics for hte evaluation. This class contains a choice of 3 profilers to collect computational metrics for evaluations: Basic, Deterministic, and Null, where the first 2 provide an average CPU time for each type of computation. 
```

Comment:
Profilers are responsible for generating benchmarking metrics for the evaluation.

Reply:
Maybe make 'where the first 2 provide an average CPU time for each type of computation.' a new sentence? Also I'm not the best with grammer.